mobify/descript
# Adding script injection

Adds the ability to inject scripts into the script containers at specific locations. 

Status: **Opened for visibility**

Reviewers: @tedtate @fractaltheory @donnielrt @mobify/a-customer-success-team 
JIRA: https://mobify.atlassian.net/browse/ADJS-40

## Changes
- added `injectScript` function
- added tests

## How to test-drive this PR
- checkout this branch
- `grunt test`
- integrate into project and see if it works!
